catalog-info.yaml

```diff
@@ -18,19 +22,97 @@ metadata:
     - title: Logstash filter for Elastic integration
       url: https://github.com/elastic/logstash-filter-elastic_integration
 spec:
+  type: tool
+  owner: group:ingest-fp
+  system: platform-ingest
+  lifecycle: production
+
+# ***********************************
+# SECTION START: Pull Request pipeline
+# ***********************************
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: logstash-filter-elastic-integration-pull-request-pipeline
+  description: 'A pipeline for Logstash filter for Elastic integration Pull Requests'
+  links:
+    - title: 'Logstash filter for Elastic integration Pull Request pipeline'
+      url: https://buildkite.com/elastic/logstash-filter-elastic-integration-pull-request-pipeline
+spec:
+  owner: group:ingest-fp
+  type: buildkite-pipeline
+  system: buildkite
   implementation:
     apiVersion: buildkite.elastic.dev/v1
     kind: Pipeline
     metadata:
-      description: ':logstash: The logstash-filter-elastic_integration :pipeline:'
-      name: logstash-filter-elastic_integration-ci
+      description: ':logstash: The logstash-filter-elastic_integration-pull-request :pipeline:'
+      name: logstash-filter-elastic_integration-pull-request
     spec:
-      pipeline_file: .buildkite/pipeline.yml
-      provider_settings:
-        build_pull_request_forks: true
+      pipeline_file: .buildkite/pull-request-pipeline.yml
       repository: elastic/logstash-filter-elastic_integration
+      provider_settings:
+        build_pull_requests: true
+        build_pull_request_forks: false
+        build_branches: false
+        build_tags: false
+        filter_enabled: true
+        filter_condition: >-
+          build.creator.name == 'elasticmachine' && build.pull_request.id != null
+      env:
+        ELASTIC_PR_COMMENTS_ENABLED: 'true'
+        ELASTIC_SLACK_NOTIFICATIONS_ENABLED: 'true'
+        SLACK_NOTIFICATIONS_CHANNEL: '#logstash-build'
+        SLACK_NOTIFICATIONS_ON_SUCCESS: 'false'
       teams:
         ingest-fp:
-          access_level: BUILD_AND_READ
-  owner: group:ingestion-team
+          access_level: MANAGE_BUILD_AND_READ
+        everyone:
```

Comment:
@roaksoax does it make sense to add this job under the [logstash buildkite organization](https://buildkite.com/elastic?team=logstash)? If yes, we should add here:

```
        logstash:
          access_level: MANAGE_BUILD_AND_READ
```

While at it, let's explicitly add rights to the `ingest-eng-prod` org too with:

```
        ingest-eng-prod:
          access_level: MANAGE_BUILD_AND_READ
```

Reply:
adding this to `logstash` org under BK sounds good to me too.